### PR TITLE
2021.2/universal/case 1379188 particle unlit blend

### PR DESCRIPTION
**Pull request by eh-unity**
### Purpose of this PR
Custom backport of particle unlit shader blending fixes.
Case: 1379188

In master fixed by this PR: https://github.com/Unity-Technologies/Graphics/pull/5762 and additionally this https://github.com/Unity-Technologies/Graphics/pull/6861

---
### Testing status
Ran local foundation tests.

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Review by phi-lira — APPROVED** [review]
It would be nice to refer to the original PR, even when this is a custom backport. It helps add context and reviewers.

**Comment by phi-lira** [comment]
Referencing the original PR is also good because then we would be able to see which PR backported it, also you should add backported tag in the original PR.

**Comment by eh-unity** [comment]
This does not have a parent PR. In master the issue was fixed by the overall blending changes, while here I fixed only the reported issue. And while doing so I discovered another issue which is fixed in another PR in master.
But I can add reference to the blending PR in master.

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Version Updated
 The version number for this package has increased due to a version update of a related graphics package.
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/CHANGELOG.md:11]: You can remove this part since you're the first one adding something to 12.1.5.

**Review by ellioman — APPROVED** [review]
Just fix the changelog and we're good.

**Review by simon-engelbrecht-soerensen — APPROVED** [review]

**Comment by eh-unity** [comment]
Test failures in commit https://github.com/Unity-Technologies/Graphics/pull/6840/commits/ad77d5b7a80f862039eb5b93ada05b6b84f41685
appear to be the same as in 21.2/staging.

**Pull request merged**